### ISSUE TITLE
Tone.BufferSource: buffer is either not set or not loaded

### ISSUE THREAD

**Issue by danicuki** [googlecreativelab/inside-music#5]
I set up everything as described. When I click on my own song to play, I get this message:

```
   Main.js:1 Uncaught Error: Tone.BufferSource: buffer is either not set or not loaded.
    at t.BufferSource.start (Main.js:1)
    at t.Player._start (Main.js:7)
    at t.Player.t.Source.start (Main.js:1)
    at e.value (Main.js:1)
    at HTMLElement.<anonymous> (Main.js:1)
    at Object.t.exports.fireEvent (vendor.bundle.js:14)
    at vendor.bundle.js:14
    at Array.map (<anonymous>)
    at HTMLElement.value (vendor.bundle.js:14)
    at HTMLButtonElement.e (Main.js:1)
```

**Comment by sonicviz** [comment]
Check the vo files, for loading etc. If you F12 you'll probably see some errors. I had the same issue but fixed it with those http://www.sonicviz.com/wp/project/spatial-music/

**Comment by danicuki** [comment]
In fact, there are many missing files. We should improve documentation, to instruct what files should be put into /audio/vo directory.

**Comment by tambien** [comment]
Yeah, as we mentioned in the REAMDE, this is entirely without audio files. So if you want to get it running locally, you'll need to add files and change a few lines in the code. 

One of the missing files is for [this line](https://github.com/googlecreativelab/inside-music/blob/master/src/splash/Scene.js#L38) which creates the audio-reactive visuals in the splash page. You can replace that with any file since it's never heard. 

The rest of the VO files are loaded from [here](https://github.com/googlecreativelab/inside-music/blob/master/src/sound/Voice.js) and can be either commented out or changed in the config file under the key "intro".
